hammer/technology/sky130/hack_iquantus_hook.py

```diff
@@ -0,0 +1,17 @@
+import importlib
+from hammer.vlsi import CLIDriver, HammerTool, HammerToolHookAction
+
+# to be inserted post- route_design
+def hack_iquantus(x: HammerTool) -> bool:
```

Comment:
Is it normal to define an entirely new Python file for a hook like this? I feel like most of the hooks are just defined in the __init__.py file.

Reply:
yeah I can do that